components/count-badge/count-badge-icon.js

```diff
@@ -29,37 +29,35 @@ class CountBadgeIcon extends CountBadgeMixin(LitElement) {
 			outline: none;
 		}
 
-		.d2l-count-badge-wrapper {
-			display: inline-block;
-		}
-
 		:host {
-			padding-right: var(--d2l-count-badge-icon-padding);
-		}
-
-		:host([dir="rtl"]) {
-			padding-left: var(--d2l-count-badge-icon-padding);
-			padding-right: 0;
-		}
-
-		:host([size="large"]) {
-			--d2l-count-badge-icon-padding: calc(var(--d2l-count-badge-icon-height) - 0.7rem);
-			margin-top: -0.7rem;
-		}
-
-		:host([size="small"]) {
-			--d2l-count-badge-icon-padding: calc(var(--d2l-count-badge-icon-height) - 0.55rem);
-			margin-top: -0.55rem;
+			display: inline-block;
+			padding-bottom: 0.2rem;
+			padding-left: 0.5rem;
+			padding-right: 0.5rem;
+			position: relative;
 		}
 
 		:host([icon*="tier1:"]) {
 			--d2l-count-badge-icon-height: 18px;
+			--d2l-count-badge-icon-padding: calc(-50% + 11px);
 		}
 		:host([icon*="tier2:"]) {
 			--d2l-count-badge-icon-height: 24px;
+			--d2l-count-badge-icon-padding: calc(-50% + 14px);
```

Comment:
This fun math tells us the x transform value to center the count badge over the bubble (used for long numbers beyond 3 digits)